shared/utils/relabel-ids.py

```diff
@@ -339,16 +330,53 @@ def verify_correct_form_of_referenced_cce_identifiers(xccdftree):
                 identcce.getparent().remove(identcce)
 
 
-def main():
-    if len(sys.argv) < 3:
-        sys.stderr.write("Provide an XCCDF file and an ID name scheme.\n")
-        sys.stderr.write("This script finds check-content files (currently, "
-                         "OVAL and OCIL) referenced from XCCDF and "
-                         "synchronizes all IDs.\n")
-        sys.exit(1)
+def create_parser():
+    parser = argparse.ArgumentParser(
+        description="This script finds check-content files (currently, "
+        "OVAL and OCIL) referenced from XCCDF and synchronizes all IDs.")
+    parser.add_argument("xccdf_file")
+    parser.add_argument("id_name", help="ID naming scheme")
+    return parser
+
 
-    xccdffile = sys.argv[1]
-    idname = sys.argv[2]
+def checkcontent_hrefarttr_startswith_http_https(check_content_ref):
```

Comment:
super long function name

Reply:
Yes, but do you have any suggestions how to change it? I am somehow OK with it.

Reply:
`check_content_href_is_remote`?

Reply:
If you insist on the long name at least remove the typo ;-)